### ISSUE TITLE
Use "INVNUM" and "DESC" params for reference transactions

### ISSUE THREAD

**Issue by tiagopog** [paypal/merchant-sdk-ruby#31]
Hey there!

When doing a reference transaction via PayPal's NVP API I can manage to use the `INVNUM` and `DESC` params with ease (see the end of the `curl`'s data param):

```
~ curl -X POST https://api-3t.sandbox.paypal.com/nvp --data 'USER=xxx&PWD=yyy&SIGNATURE=zzz&METHOD=DoReferenceTransaction&VERSION=114.0&PAYMENTACTION=SALE&AMT=42.00&CURRENCYCODE=BRL&REFERENCEID=B%2d7JD95025TF788012H&INVNUM=123456789&DESC=Awesome+Product'
```

And it answers properly:

```
TIMESTAMP=2016%2d09%2d15T20%3a10%3a31Z&CORRELATIONID=42420eee4b40e...
```
I couldn't find any explicit reference to the use of these params on the gem's docs/specs. Hope to hear from you guys if it's possible to use such params. In case it isn't, shall I send a PR?

**Comment by tiagopog** [comment]
Just figured out that it doesn't really load those params: https://github.com/paypal/merchant-sdk-ruby/blob/master/lib/paypal-sdk/merchant/data_types.rb#L2409

**Comment by tiagopog** [comment]
I achieved the expected request body after reading the [documentation around the SOAP API](https://developer.paypal.com/docs/classic/api/merchant/DoReferenceTransaction_API_Operation_SOAP/).

It turns out that both params need to be sent within the `PaymentDetails` member and they both have different names: `INVNUM` => `InvoiceID`; `DESC` => `OrderDescription`.

Now I got things working properly, I hope the gem's documentation/samples get improved soon.
